Sources/NIOWebSocket/WebSocketFrame.swift

```diff
@@ -71,6 +72,27 @@ extension WebSocketMaskingKey: ExpressibleByArrayLiteral {
     }
 }
 
+extension WebSocketMaskingKey {
+    /// Returns a random masking key, using the given generator as a source for randomness.
+    /// - Parameter generator: The random number generator to use when creating the
+    ///     new random masking key.
+    /// - Returns: A random masking key
+    @inlinable
+    public static func random<T>(
+        using generator: inout T
+    ) -> Self where T: RandomNumberGenerator {
+        WebSocketMaskingKey(networkRepresentation: .random(in: UInt32.min...UInt32.max, using: &generator))
+    }
+    
+    /// Returns a random masking key, using the `SystemRandomNumberGenerator` as a source for randomness.
+    /// - Returns: A random masking key
+    @inlinable
+    public static func random() -> Self {
```

Comment:
```suggestion
    public static func random() -> WebSocketFrame {
```

I think 5.0 requires this